xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -214,7 +214,8 @@ We recommend that you leave this value unchanged unless changing it makes an obs
         <ReturnType>System.Text.Json.Serialization.JsonIgnoreCondition</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Specifies a condition to determine when properties with default values are ignored during serialization or deserialization.
+ The default value is <see cref="F:System.Text.Json.Serialization.JsonIgnoreCondition.Never" />.</summary>
         <value>To be added.</value>
         <remarks>To be added.</remarks>
       </Docs>
```

Comment:
I cannot comment on the proper tag but, can we add the following to the remarks section of `JsonSerializerOptions.ReferenceHandler`?

> By default, serialization does not support objects with cycles and does not preserve duplicate references. Metadata properties will not be written when serializing reference types and will be treated as regular properties on deserialize.
> 
> * On Serialize:
>   * Treats duplicate object references as if they were unique and writes all their properties.
>   * The serializer throws a <xref:System.Text.Json.JsonException> if an object contains a cycle.
> * On Deserialize:
>   * Metadata properties (`$id`, `$values`, and `$ref`) will not be consumed and therefore will be treated as regular JSON properties.
>   * The metadata properties can map to a real property on the returned object if the property names match, or will be added to the <xref:System.Text.Json.Serialization.JsonExtensionDataAttribute> overflow dictionary, if one exists; otherwise, they are ignored.
> 
> Use <xref:System.Text.Json.Serialization.ReferenceHandler.Preserve> to enable unique object reference preservation on serialization and metadata consumption to read preserved references on deserialization.

Reply:
I applied your text, please review the change.